src/js/components/molecules/cards/card-article/card-article.ts

```diff
@@ -9,8 +9,29 @@ import WithRender from './card-article.vue.html';
 @Component({
     name: 'CardArticle',
 })
-
 export class CardArticle extends CardBase implements CardArticleProps {
-    @Prop({ type: String }) description?: CardArticleProps['description'];
-    @Prop({ type: String }) url?: CardArticleProps['url'];
+    @Prop({ type: String }) description!: CardArticleProps['description'];
+    @Prop({ type: Array }) labels?: CardArticleProps['labels'];
+    @Prop({ type: String }) url!: CardArticleProps['url'];
+
+    get listItems(): object {
+        let properties: any = [];
+
+        if (this.labels) {
```

Comment:
with a default empty array, this check becomes unnecessary

Reply:
can't figure out how to get typescript to agree.